packages/material-ui-benchmark/package.json

```diff
@@ -32,6 +32,6 @@
     "nodemod": "^1.5.19",
     "react": "^16.3.0",
     "react-jss": "^10.0.0-alpha.23",
-    "styled-system": "latest"
+    "styled-system": "^5.1.0"
```

Comment:
:+1: for specifying the version here, otherwise the benchmark results can differ depending on when `yarn install` was ran